indico/modules/events/contributions/templates/display/contribution_display.html

```diff
@@ -216,6 +217,17 @@ <h3>
 
     {{ render_attachments(contribution) }}
 
+    {% if contribution.references -%}
+        <section>
+            <div class="header">
+                <div class="header-row">
+                    <h3>{% trans %}External References{% endtrans %}</h3>
```

Comment:
```suggestion
                    <h3>{% trans %}External references{% endtrans %}</h3>
```

Reply:
Should I also change `Presentation Materials` to `Presentation materials` while I'm at it?

Reply:
good idea :) i checked the authors where it's also not all capitalized

Reply:
`Presentation Materials` already has translations. Should I update them too?

Reply:
No, they are managed via transifex so updating them in the repo wouldn't help. Anyway, translators will have to make updates for 3.0 anyway, and they'll probably get their old translation suggested because of how minor this change is.